code/datums/status_effects/debuffs/revival_shock.dm

```diff
@@ -0,0 +1,18 @@
+/datum/status_effect/revival_shock
+	id = "revival_shock"
+	alert_type = /atom/movable/screen/alert/status_effect/revival_shock
+	duration = 10 MINUTES
+	status_type = STATUS_EFFECT_REFRESH
```

Comment:
```suggestion
	status_type = STATUS_EFFECT_REFRESH
	remove_on_fullheal = TRUE
	heal_flag_necessary = HEAL_ADMIN // unless you want it on generic HEAL_STATUS. up to you
	
```

Reply:
Also you can put the moodlet application on the station effect apply / remove so ahealing removes the moodlet as well